File: lib/src/main/java/com/darklycoder/wifitool/lib/info/WiFiCreateConfigStatusInfo.java
package com.darklycoder.wifitool.lib.info;

import android.net.wifi.WifiConfiguration;

/**
 * 创建WiFi配置状态
 */
public class WiFiCreateConfigStatusInfo {

    public String SSID;
    public WifiConfiguration configuration;
    public boolean isSuccess;

    public boolean isSuccess() {
        return isSuccess && null != configuration;
    }

}

File: lib/src/main/java/com/darklycoder/wifitool/lib/info/WiFiCreateConfigStatusInfo.kt
package com.darklycoder.wifitool.lib.info

import android.net.wifi.WifiConfiguration

/**
 * 创建WiFi配置状态
 */
data class WiFiCreateConfigStatusInfo(
        var SSID: String? = null,
        var configuration: WifiConfiguration? = null,
        var isSuccess: Boolean = false
)